packages/manager/apps/pci-databases-analytics/src/pages/services/[serviceId]/users/_components/ToggleAcl.component.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from 'react';
```

Comment:
Same as `DeleteIndexModal` component, with `strictNullChecks` on, `service` & `projectId` are possibly `undefined` and can lead to runtime errors